Laba2 kapustina

PolinaKapustina opened the pull request.

PolinaKapustina committed:
Add files via upload

PolinaKapustina committed:
Add files via upload

ValentinV95 commented:
Отчет принят. Код принят с замечаниями:
1) Есть опечатка, из-за чего код не компилируется
2) Отсутствие генерации, не дает возможности проверить работу на больших системах
3) Матрица наследует операторы от вектора, но для не они корректно работать будут не все
4) Есть схожие уникальные черты кода с кодом, который я проверял ранее

PolinaKapustina committed:
Add files via upload

Если ничего не напутала с файлами, то должна была быть добавлена возможность заполнения матрицы рандомными числами, а так же исправлена ошибка из-за которой программа не компилировалась.

PolinaKapustina commented:
Замечания учтены. Исправлена опечатка из-за которой код не компилировался. Добавлена возможность заполнения матрицы рандомными элементами. Измененные файлы загружены с приставкой _new после названия.

PolinaKapustina committed:
Update SecondLaba_new.cpp

ValentinV95 commented:
Исправления приняты
